### DISCUSSION TITLE
fatal: failed to expand user dir in: '~~.gitignore_global'

### DISCUSSION THREAD

**Discussion by icaroh1** [community/community#50735] [Other Feature Feedback, Questions, & Ideas]
Hi, i use Mac 13.2.1, and im trying in terminal to solve the problem with .DS_Store files...

how to solve this definitely?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @icaroh1 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by icaroh1** [reply]
Thanks!
